api/envoy/type/matcher/string.proto

```diff
@@ -64,6 +64,11 @@ message StringMatcher {
     // The input string must match the regular expression specified here.
     RegexMatcher safe_regex = 5 [(validate.rules).message = {required: true}];
   }
+
+  // If true, indicates the exact/prefix/suffix matching should be case insensitive. This has no
+  // effect for the safe_regex match.
+  // For example, the matcher *data* will match both input string *Data* and *data* if set to true.
+  bool ignore_case = 6;
```

Comment:
Can we have a description in the comments about whether this field is true/false by default?

Reply:
All `bool` fields are false by default in protobuf. See: https://developers.google.com/protocol-buffers/docs/proto3#default

Reply:
Thank you!